### ISSUE TITLE
ActiveSupport.on_load not being called

### ISSUE THREAD

**Issue by chriscz** [palkan/active_event_store#4]
## What did you do?
I set up `active_event_store` according to the instructions:

```ruby
ActiveSupport.on_load :active_event_store do |store|
  store.subscribe MyEventHandler, to: ProfileCreated
end
```

## What did you expect to happen?
I expected the `on_load` block to be executed.

## What actually happened?
The block doesn't execute

## Additional context
- When using a `to_prepare` block on the application `ActiveEventStore.event_store` has been initialized.
- MWE: https://github.com/chriscz/aes-loading-bug

## Environment

**Ruby Version:** 2.5.8

**Framework Version (Rails, whatever):** Rails 5.2.4.4

**Active Event Store Version:** 0.2.0

**Comment by palkan** [comment]
Thanks for the reproduction! Checking now.

**Comment by palkan** [comment]
Fixed and released in 0.2.1
